common/tier4_debug_rviz_plugin/src/jsk_overlay_utils.cpp

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) 2014, JSK Lab
```

Comment:
This file is included in some packages in autoware.universe.
This should be refactored with another PR.

https://github.com/autowarefoundation/autoware.universe/blob/main/common/tier4_planning_rviz_plugin/src/tools/jsk_overlay_utils.cpp
https://github.com/autowarefoundation/autoware.universe/blob/main/common/tier4_vehicle_rviz_plugin/src/tools/jsk_overlay_utils.cpp